spec/models/miq_ae_datastore_spec.rb

```diff
@@ -94,8 +94,20 @@ def setup_version_xml(v)
     expect(MiqAeMethod.count).to         eq(3)
   end
 
-  it ".default_domain_names" do
-    expect(MiqAeDatastore.default_domain_names).to include("ManageIQ")
+  describe "#default_domain_names" do
+    context "vmdb plugin domains" do
```

Comment:
I think the `context` & `it ".default_domain_names"` is redundant, should the context be removed and the `it` changed to `vmdb plugin domains`?